apps/core/src/dashboards/dashboards.service.ts

```diff
@@ -29,4 +29,9 @@ export class DashboardsService {
   public async delete(id: Dashboard['id']) {
     return this.repository.delete(id);
   }
+
+  public async bulkDelete(ids: Dashboard['id'][]) {
+    // TODO: use BatchWriteItem instead of Promise.all
+    return Promise.all(ids.map((id) => this.repository.delete(id)));
```

Comment:
I don't think [Promise.all(...)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) can handle multiple rejections
> The rejection reason is the rejection reason of the first promise that was rejected.

is that the desirable behavior or [Promise.allSettled()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled)'s behavior would serve better?